kahuna/public/js/search/view.html

```diff
@@ -5,14 +5,15 @@
     </gr-top-bar-nav>
 
     <gr-top-bar-actions>
```

Comment:
Might be better to have the `ng-if` on this element rather than each individual child element?

Reply:
If used at the gr-top-bar-actions element, we lose the three dots menu with the Quotas, Feedback, Logout options